extremeflash/ws_ap3935i.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python3
+#
+#     Copyright (C) 2023 Grische
```

Comment:
You can update the Copyright to your own. As long as the license stays the same.